pkg/controller/provisionmanager/provisionmanager_controller_test.go

```diff
@@ -164,8 +163,9 @@ func TestProvisionManagerController(t *testing.T) {
 		require.NoError(t, core.SchemeBuilder.AddToScheme(scheme), "Failed core.SchemeBuilder.AddToScheme()")
 		require.NoError(t, apps.SchemeBuilder.AddToScheme(scheme), "Failed apps.SchemeBuilder.AddToScheme()")
 		pmr := newProvisionManager()
```

Comment:
Should it be used if you define service already?

Reply:
Turns out that in this test pmr is not necessary.